tools/adios2pio-nm/adios2pio-nm-lib.cxx

```diff
@@ -2252,6 +2253,26 @@ int ConvertBPFile(const string &infilepath, const string &outfilename,
             return BP2PIO_ERROR;
         }
 
+        /* Get the value of /__pio__/info/decomp_stored flag */
+        adios2::Variable<int> bpDecompStored = bpIO[0].InquireVariable<int>("/__pio__/info/decomp_stored");
+        if (bpDecompStored)
+        {
+            bpReader[0].Get(bpDecompStored, &decomp_stored_flag, adios2::Mode::Sync);
```

Comment:
Do we have to handle the error case here or would an exception thrown by ADIOS be enough?

Reply:
It seems that the Get method returns void and an exception might be thrown.
```
    /**
     * Get data associated with a Variable from the Engine
     * @param variable contains variable metadata information
     * @param data user data to be associated with a variable, it must be
     * pre-allocated
     * @param launch mode policy
     * <pre>
     * 		Mode::Deferred, lazy evaluation, do not use data until
     * first PerformGets, EndStep, or Close. This is the preferred way.
     * 		Mode::Sync, data is obtained by the Engine and can be used
     * immediately.
     * Special case, only use if necessary.
     * </pre>
     * @exception std::invalid_argument for invalid variable or nullptr data
     */
    template <class T>
    void Get(Variable<T> variable, T *data, const Mode launch = Mode::Deferred);

    void Get(VariableNT &variable, void *data, const Mode launch = Mode::Deferred);
```